src/int.js

```diff
@@ -121,18 +128,22 @@ Sk.builtin.int_ = function (x, base) {
     return this;
 };
 
+Sk.abstr.setUpInheritance("int", Sk.builtin.int_, Sk.builtin.numtype);
+
 Sk.builtin.int_.co_varnames = [ "base" ];
 Sk.builtin.int_.co_numargs = 2;
-Sk.builtin.int_.$defaults = [ new Sk.builtin.int_(10) ];
```

Comment:
I moved this value to [constants.js](https://github.com/mchat/skulpt/blob/subclass/src/constants.js#L21). All the builtin constants are initialized after all the builtin classes have been defined.